spaces/S000017/properties/P000189.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000017
+property: P000189
+value: true
+---
+
+Since closed subsets properly contained in $X$ are countable, if $X$ were $\sigma$-connected then it would be a countable union of countable sets, and so countable. But {S17|P65}.
```

Comment:
```suggestion
Since closed subsets properly contained in $X$ are countable, if $X$ were not $\sigma$-connected then it would be a countable union of countable sets, and so countable. But {S17|P65}.
```